package.json

```diff
@@ -11,7 +11,8 @@
     "format": "prettier --write . && yarn lint --fix",
     "test": "vitest",
     "coverage": "vitest run --coverage",
-    "test:e2e": "EXTENSION=keplr synpress run --configFile=tests/e2e/synpress.config.cjs"
+    "test:e2e": "EXTENSION=keplr synpress run --configFile=tests/e2e/synpress.config.cjs",
+    "test:e2e:ci": "start-server-and-test 'yarn dev' http-get://localhost:5173 'yarn test:e2e'"
```

Comment:
is this only for CI? seems useful locally too

Reply:
locally, you'd want to start the dapp + agoric local chain so this doesn't exactly cover all of the things you'd want to start up before testing.
And most of the time while developing, those things are already booted up so its best to have a command that only starts the tests.

but if you think its worthwhile, I'll remove the `test:e2e` command and rename `test:e2e:ci` to `test:e2e`

Reply:
I agree we shouldn't assume the developer hasn't already started Docker, so I think there should be two commands. I suggest renaming `test:e2e:ci` to what it does instead of assuming where it will be used. E.g. `test:e2e-with-server`.

But the more I think about it, I'm not comfortable with mixing e2e and unit tests in `test`. Up til now everything in `test` has run with the `yarn test` command. Now `yarn test` skips `tests/e2e` and there are two new commands to run the e2e tests.

However I think this should be consistent across repos and has more stakeholders, so go ahead and merge whatever makes sense to you and the developers of these repos can adjust later. cc @samsiegart @0xpatrickdev